src/bundler.ts

```diff
@@ -130,6 +134,37 @@ export class Bundler {
   }
 
   /**
+   * Generates a BundleManifest with all bundles defined, using entrypoints,
+   * strategy and mapper.
+   *
+   * @param {Array<string>} entrypoints The list of entrypoints that will be
```

Comment:
Recommend against including a type in the jsdoc. It's already in the typescript, and the typescript is checked, this is not. Can just leave it out and it's still legal jsdoc (I think).